eoscostarica/eos-rate
# Change home page video

### GH Issue #62

#

### Steps to test

1. go to EOS Rate home page
2. scroll down and see the new video

### Checklist
